src/public/modules/core/components/banner.client.component.js

```diff
@@ -26,4 +33,21 @@ function bannerController() {
       vm.bannerHidden = true
     })
   }
+
+  const processBannerMessage = () => {
+    // Retrieve banenr type from message, but it is possible that no types
+    // exist.
+    const vmMessage = vm.message || ''
+    const type = vmMessage.split(':').shift().toLowerCase()
```

Comment:
```suggestion
    const type = vmMessage.split(':').shift().trim().toLowerCase()
```
suggest adding `trim()` in case user enters a space before semicolon which will then return undefined

Reply:
If there's a space before the semicolon, then the prefix is invalid and will default to INFO banner. I think that's safer than changing prefixes etc. Other reviewers: thoughts?

Reply:
I'm slightly in favour of being more forgiving of human error (i.e. adding the `trim`), but no strong preference

Reply:
We are in a democracy!!! Done.